handbook/engineering/core-services/index.md

```diff
@@ -1,11 +1,66 @@
+
+Each issue is:
+
+- Closed if appropriate.
+- Commented on if it's unclear what next steps are.
+- Assigned to the up-coming milestone if deemed a priority.
+- Left in the backlog otherwise.
+
+Every issue assigned to the upcoming milestone needs to be:
+
+1. Assigned to a single owner.
+1. Estimated with one of the **estimate/Nd** labels.
+1. Categorized with any combination of these labels: **spike**, **roadmap**, **bug**, **debt**, **customer**.
+
+After this triage process, the team manager puts together a draft tracking issue (based on previous tracking issues) containing this initial proposed work for the iteration.
+
+At this point, everyone on the team is asked to:
+
+1. Update their availability for the month in the Not Working calendar and write down the number of working days in the designated section in the tracking issue.
+1. Propose other work for the iteration in a comment in the tracking issue. Each proposed work item must be captured in a separate GitHub issue that is assigned to the right owner, estimated, and categorized as per the above mentioned requirements.
+
+All planned work items are meant to be finished in the iteration. If this doesn't happen, it is either due to unknown unknowns or lack of estimating accuracy. While we cannot work on the former, we can on the latter — over time, we aim to estimate better.
+
+Estimation has a bad reputation in some circles, but we find it valuable to:
+
+1. Prevent any single team member having more work assigned than available work days.
+1. Ensure everyone has enough slack in the iteration (~20% of their time). This is to account for unexpected things, to foster creativity and to prevent burn-out.
+1. Remind the person working on the issue to avoid time sinks or approaches that would miss the estimate.
+
+Roadmap items may take more than an iteration to implement. As such please link to an issue which is scoped to this iteration. If it isn't clear how to break down a larger item in a scope increment, a **spike** of one or two days could be appropriate to determine such scope and then create the resulting issues for the iteration — it's fine to update the tracking issue with the results of a spike during the iteration.
+
+After the internal discussion of the team's work plan for the iteration ends, the tracking issue is ready to be *reviewed* by external stake-holders. The team manager asks for +1s from Product and the VP of Engineering.
+
+After adjusting the work plan according to their feedback (if necessary), the team manager marks the tracking issue as final.
+
+The tracking issue body will be frequently updated throughout this process and the iteration [with a tool](https://github.com/sourcegraph/sourcegraph/blob/master/internal/cmd/tracking-issue/main.go) that removes the need for manually updating it — team members are only required to handle the GitHub issues themselves.
```

Comment:
To the extent that this tool is used to create the tracking issue instead of just updating it, I think it would be useful to first mention this tool earlier in the document where creating the tracking issue is discussed.